setup.py

```diff
@@ -16,6 +16,7 @@
           'License :: OSI Approved :: GNU Affero General Public License v3',
           'Programming Language :: Python :: 3 :: Only'
       ],
+      python_requires=">=3.7,<3.10",
```

Comment:
@Samira-El Hi, does this PR introduce any codes that are not compatible with Python 3.10 ?

Or we add `<3.10` just because the test suite does not cover Python v3.10 yet?